abr_analyze/nengo_utils/tests/test_intercepts_scan.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import numpy as np
+
+from abr_analyze.nengo_utils import intercepts_scan
+
+def get_params():
+    n_ensembles = 1
+    n_neurons = 50
+    # encoders should be n_ensembles x n_neurons x n_dims
+    encoders = np.array([np.random.choice([-1, 1], n_neurons)[:, None]
+                         for ii in range(n_ensembles)])
+    intercept_vals = [
+        [-.5, 0, -.25],
+        [-.9, .5, .25],
+        [.5, 1, .75],
+        ]
+    input_signal = np.sin(np.linspace(0, 2*np.pi, 100))
+
+    return encoders, intercept_vals, input_signal
+
+
+
+def test_proportion_neurons_active():
+    encoders, intercept_vals, input_signal = get_params()
+
+    intercepts_scan.proportion_neurons_active(
+        encoders=encoders,
+        intercept_vals=intercept_vals,
+        input_signal=input_signal)
+
+
+def test_proportion_time_active():
+    encoders, intercept_vals, input_signal = get_params()
+
+    intercepts_scan.proportion_neurons_active(
+        encoders=encoders,
+        intercept_vals=intercept_vals,
+        input_signal=input_signal)
+
+
+def test_review(plt):
```

Comment:
What is the `plt` for?

Reply:
by doing it this way and using `pytest-plt` the plots are only generated when the `--plots` tag is added to the pytest call